interfaces/basic-auth-information.tsx

```diff
@@ -0,0 +1,5 @@
+export interface BasicAuthInformation {
+  basicAuthRequired: boolean;
```

Comment:
```suggestion
  required: boolean;
```

Should be unique enough in this context